File: Python/8. 뒤집은 소수 (중상)/AA.py
import sys
# sys.stdin=open('input.txt', 'rt')

n=int(input())
arr=list(map(int, input().split()))

def isPrime(a):
    flag=True
    for i in range(2, a):
        if a%i==0:
            flag=False
            break
    if flag:
        if a!=1: 
            print(a, end=' ')

for i in range(len(arr)):
    num=arr[i]
    s=str(num)
    temp=''
    for c in s :
        temp=c+temp
    
    reverse=int(temp)
    isPrime(reverse)

File: Javascript/섹션2. (진행중) 1-2차원 배열 탐색/3. 가위바위보 (중하)/AA.js
function solution(num, arr){
    let answer=[]
    
    for(let i=0;i<num;i++){
        if(i==0) {
            answer.push(parseInt(arr[i]))
            continue
        }
        value=parseInt(arr[i])
        lastValue=parseInt(arr[i-1])
        
        if(value>lastValue){
            answer.push(value)
        }
    }

    return answer
}
let str="130 135 148 140 145 150 150 153"
arr=str.split(' ')
console.log(solution(8, arr))